### PR TITLE
Tech CORE-461 | Rename inspr CLI as insprctl

### PR DESCRIPTION
**Pull request by franciscobonand**
# Description

Kind: Tech Pendency

Section: Inspr CLI

Summary: Modified Inspr CLI so that the main command is `insprctl` instead of `inspr`

## Changelog

> Renamed folder `cmd/inspr` to `cmd/insprctl`
> Modified commands and command descriptions in `pkg/cmd` and `cmd/insprctl`
> Fixed tests that were broker after the main command change

## Pay attention to

**ONCE THIS PR IS MERGED, REMEMBER TO REINSTALL THE CLI**

### CODE REVIEW

**Review by nicholaspcr — APPROVED** [review]

File: cmd/insprctl/cli/apply_test.go [hunk 1]
```diff
@@ -338,7 +338,7 @@ func Test_applyValidFiles(t *testing.T) {
 			},
 			want:    nil,
 			funcErr: ierrors.NewError().Message("unauthorized").Unauthorized().Build(),
-			errMsg:  "We couldn't authenticate with the cluster. Is your token configured correctly?\n",
+			errMsg:  "we couldn't authenticate with the cluster. Is your token configured correctly?\n",
```
Comment (LucasStarlingdePaulaSalles) [inline cmd/insprctl/cli/apply_test.go]: 'We' is suggestive of an action performed by a group, I would use ' Failure to authenticate ...'

File: cmd/insprctl/cli/cluster.go [hunk 2]
```diff
@@ -13,19 +13,19 @@ import (
 func NewClusterCommand() *cobra.Command {
 	getBrokers := cmd.NewCmd("brokers").
 		WithDescription("Retrieves brokers currently installed").
-		WithExample("get cluster's brokers", "inspr cluster brokers").
+		WithExample("get cluster's brokers", "cluster brokers").
```
Comment (LucasStarlingdePaulaSalles) [inline cmd/insprctl/cli/cluster.go:16]: missing 'insprctl' on command examble

File: cmd/insprctl/cli/cluster.go [hunk 3]
```diff
@@ -13,19 +13,19 @@ import (
...
 		WithAliases("b").
 		NoArgs(getBrokers)
 	authInit := cmd.NewCmd("init").
 		WithDescription("Init configures insprd's default token").
-		WithExample("init insprd as admin", " inspr cluster init <admin_password>").
+		WithExample("init insprd as admin", "cluster init <admin_password>").
```
Comment (LucasStarlingdePaulaSalles) [inline cmd/insprctl/cli/cluster.go:21]: missing 'insprctl' on command examble

File: cmd/insprctl/cli/cluster.go [hunk 4]
```diff
@@ -13,19 +13,19 @@ import (
...
 		WithCommonFlags().
 		ExactArgs(1, authInit)
 	return cmd.NewCmd("cluster").
 		WithDescription("Configure aspects of your inspr cluster").
 		WithLongDescription("Cluster takes a subcommand of (brokers | init)").
-		WithExample("get cluster's brokers", "inspr cluster brokers").
-		WithExample("init insprd as admin", " inspr cluster init <admin_password>").
+		WithExample("get cluster's brokers", "cluster brokers").
+		WithExample("init insprd as admin", "cluster init <admin_password>").
```
Comment (LucasStarlingdePaulaSalles) [inline cmd/insprctl/cli/cluster.go:28]: missing 'insprctl' on command examble

File: pkg/cmd/utils/cli_client.go [hunk 5]
```diff
@@ -81,7 +81,7 @@ func RequestErrorMessage(err error, w io.Writer) {
 	if ok {
 		switch ierr.Code {
 		case ierrors.Unauthorized:
-			fmt.Fprintf(w, "We couldn't authenticate with the cluster. Is your token configured correctly?\n")
+			fmt.Fprintf(w, "we couldn't authenticate with the cluster. Is your token configured correctly?\n")
```
Comment (LucasStarlingdePaulaSalles) [inline pkg/cmd/utils/cli_client.go]: Same thing here

File: pkg/cmd/utils/cli_client_test.go [hunk 6]
```diff
@@ -201,7 +201,7 @@ func TestRequestErrorMessage(t *testing.T) {
 			args: args{
 				ierrors.NewError().Unauthorized().Build(),
 			},
-			wantW: "We couldn't authenticate with the cluster. Is your token configured correctly?\n",
+			wantW: "we couldn't authenticate with the cluster. Is your token configured correctly?\n",
```
Comment (LucasStarlingdePaulaSalles) [inline pkg/cmd/utils/cli_client_test.go]: Same her for testing

**Review by LucasStarlingdePaulaSalles — CHANGES_REQUESTED** [review]

**Review by LucasStarlingdePaulaSalles — APPROVED** [review]

File: cmd/insprctl/cli/apply_test.go (hunk 1 above)
Comment (franciscobonand) [reply cmd/insprctl/cli/apply_test.go]: You're right

**Pull request merged**